tests/forms/FormFieldTest.php

```diff
@@ -3,7 +3,52 @@
  * @package framework
  * @subpackage tests
  */
-class FormFieldTest extends SapphireTest implements TestOnly {
+class FormFieldTest extends SapphireTest {
+
+	public function testDefaultClasses() {
+		Config::nest();
+
+		Config::inst()->update('FormField', 'default_classes', array(
+			'class1',
+		));
+
+		$field = new FormField('MyField');
+
+		$this->assertContains('class1', $field->extraClass(), 'Default class not added to class list');
+
```

Comment:
The final parameter is the message that is shown when the assertion fails. In this example that would be when the expected class "class1" is no in the string returned by the function.

If the assertion failed, you're suggesting the message should be "Default class is added to class list", which is not why the test failed.

If no final argument is provided, then it would use the default which would be: "Failed to assert that string "[insert returned string]" contained "class1".